scripts/drush/verify-pagelayer-data.php

```diff
@@ -0,0 +1,238 @@
+#!/usr/bin/env drush
+<?php
+
+use Drush\Drush;
+
+/*
+ * Can pass as args:
+ * --sample if set, samples up to 30 paths of each content type+language combo
+ * --file <filename> Complete path to CSV file, or path relative to /var/www/web
+ */
+[$csv, $samplePaths] = parse_args($extra);
+
+if (!$csv) {
+  Drush::output()->writeln("<error>Can't read or find CSV file.</error>");
+  exit(1);
+}
+
+Drush::output()->writeln("<info>Reading CSV file.</info>");
+$counts = [];
+foreach (read_csv($csv) as $line) {
+  if ($line->pageID == 5) {
```

Comment:
You know, this entry is been bugging me and I'm going to figure out where it comes from. But it's absolutely out of scope for this refactor.